otx/algorithms/segmentation/adapters/mmseg/datasets/pipelines/loads.py

```diff
@@ -29,6 +29,11 @@
 class LoadImageFromOTXDataset(load_image_base.LoadImageFromOTXDataset):
     """Pipeline element that loads an image from a OTX Dataset on the fly."""
 
+    @check_input_parameters_type()
+    def __init__(self, to_float32: bool = False, use_otx_adapter: bool = True):
+        self.use_otx_adapter = use_otx_adapter
```

Comment:
Does image load need use_otx_adapter parameter? I suggest keep this parameter only in `LoadAnnotationFromOTXDataset`

Reply:
Unfortunately, for **test** dataset we can control this parameter only via LoadImageFromOTXDataset since we don't use LoadAnnotationFromOTXDataset there